RecoMuon/GlobalTrackingTools/python/GlobalMuonRefitter_cff.py

```diff
@@ -26,13 +26,21 @@
     RefitDirection = cms.string('insideOut'),
     PropDirForCosmics = cms.bool(False),
     RefitRPCHits = cms.bool(True),
- 
+
     # DYT stuff
     DYTthrs = cms.vint32(10, 10),
     DYTselector = cms.int32(1),
     DYTupdator = cms.bool(True),
     DYTuseAPE = cms.bool(False),
-    
+    ## Parameters for DYT threshold parametrization
+    DYTuseThrsParametrization = cms.bool(False),
+    DYTthrsParameters = cms.PSet(
+                                  eta0p8 = cms.vdouble(1, -0.919853, 0.990742),
+                                  eta1p2 = cms.vdouble(1, -0.897354, 0.987738),
+                                  eta2p0 = cms.vdouble(4, -0.986855, 0.998516),
+                                  eta2p2 = cms.vdouble(1, -0.940342, 0.992955),
+                                  eta2p4 = cms.vdouble(1, -0.947633, 0.993762),
```

Comment:
instead of copy-pasting here and in GlobalTrajectoryBuilderCommon_cff.py,
better define these parameters in a separate file and load from there.